random_menu_lunch.rb

```diff
@@ -0,0 +1,13 @@
+puts "~Le Dinner Menu~"
+
+## three arrays for describing menu items
+adjective = ["fatty", "meaty", "decadent", "skinny",  "prickly", "room-temperature", "sultry", "heated", "dry", "moist"]
+cooking_method = ["fried", "impaled", "battered", "beaten", "tenderized", "smashed", "torched", "live", "infused", "vegan"]
+food = ["snake scales", "pandas", "Hagrid beards", "piglets", "tofurkey", "whale steak", "unicorn horns", "leprechauns", "pikachus", "walruses"]
+
+# I used a while loop to output meals equal to any number of elements in an array starting at 1
+meal_count = 1
+while meal_count <= adjective.length
```

Comment:
You could augment your code to use something like `[adjective.length, cooking_method.length, food.length].min` to get the smallest length from all of your lists to ensure that you aren't going too far in your loop if the lengths of each array are different.